### DISCUSSION TITLE
iOS平台不能发送文件夹

### DISCUSSION THREAD

**Discussion by Vinb1900** [localsend/localsend#835] [L-简体中文]
iPhone发送文件夹显示没有授予必要的权限，但是进入软件设置以及软件内部设置均没有授予权限的开关，iOS版本：17.2，机型：iPhone 12

**Comment by diyfixtool** [comment]
Go into the software settings and software internal settings to see if there is a switch to grant permissions.

**Comment by Seraphxy** [comment]
情况是这样：
1）同一局域网中iPhone手机端可以发现PC，也可以传输文件，
2）但是PC电脑端无法找到iPhone设备，输ip也不行，电脑端端口都是打开得。
电脑端可以通过地址分享传输文件，但很不方便。
条件1 已经说明可以互传了。

**Comment by songofsilver** [comment]
在文件中把你要传送的文件夹移动到localsend目录下，就可以正常传送了。

**Reply by MTcool** [reply]
可行👍，多谢🙏

**Reply by gdye725gsu-lang** [reply]
非常感谢！！！

**Reply by lylc0000** [reply]
> 在文件中把你要传送的文件夹移动到localsend目录下，就可以正常传送了。

只能这样吗？目前iOS没办法任意选择文件夹吗？

**Reply by lylc0000** [reply]
好了，我发现LanDrop可以选择任意文件夹，所以开发团队还是赶紧想想办法吧。毕竟这是23年的问题了，到现在还没修好。

**Reply by AetherWish** [reply]
方法可行，非常感谢！

**Comment by flamenx** [comment]
我这边是iOS找不到电脑 显示有权限未开放
